Added support of parameters to Copy Query As -> Get-Swis-Data command

seekerer opened the pull request:
Related to [**this**](https://github.com/solarwinds/OrionSDK/issues/145) issue. Changed logic so that parameters also get passed to Get-Swis-Data Powershell command. Haven't implemented this for CURL because as far as I aware, SWIS doesn't allow execute query and pass parameters via REST API endpoint (``https://[swis-address]:17778/SolarWinds/InformationService/v3/Json/Query?query=[query]``).

tdanner commented on Src/SwqlStudio/Utils/CommandLineGenerator.cs:
```diff
@@ -7,25 +9,45 @@ namespace SwqlStudio.Utils
+            {
+                decimal tempNum = 0;
+                currentParameterIndex++;
+                var isNumber = decimal.TryParse(param.Value.ToString(), out tempNum);
```
You are not actually using `isNumber` here. I think it was probably intended to be used to skip wrapping the value in quotes on the next line, but SWIS doesn't really care. It will coerce the value from a string to a number if it needs to.

tdanner commented on Src/SwqlStudio/Utils/CommandLineGenerator.cs:
```diff
@@ -7,25 +9,45 @@ namespace SwqlStudio.Utils
+                var format = "{0}='{1}'";
+                var isLastElement = currentParameterIndex == parametersCount;
+                format = isLastElement ? format : string.Concat(format, ";");
+                result.Append(string.Format(format, param.Key, param.Value));
```
Rather than mess around with `StringBuilder` and `.Append` and conditionally inserting `;`, I would just format each parameter separately and combine them all with `string.Join`. There are never going to be enough parameters for the performance difference to matter.

tdanner commented:
Thanks! And the REST API does support query parameters, but you need to use POST instead of GET: https://github.com/solarwinds/OrionSDK/wiki/REST#query-with-parameters-request

seekerer commented:
Update: refactored based on previous comments.

Additionally, added propagation of query parameters when copying query as CURL command (Windows and Bash). 

The only thing I'm not certain is whether or not we should keep ``--insecure`` flag. It explicitly tells CURL to ignore SSL Certificate validation errors. Potentially it allows to perform man-in-the-middle attack, but I would rather keep it considering the fact SWIS is only accessible on the local network.
